diff --git a/dist/adapter/aws-lambda/handler.js b/dist/adapter/aws-lambda/handler.js
index bac308f9f67dfeacac07e9752a506f28e0000bdf..19a429cf4e8fe550373b28545bacc76990165681 100644
--- a/dist/adapter/aws-lambda/handler.js
+++ b/dist/adapter/aws-lambda/handler.js
@@ -254,12 +254,12 @@ var getProcessor = (event) => {
 };
 var isProxyEventALB = (event) => {
   if (event.requestContext) {
-    return Object.hasOwn(event.requestContext, "elb");
+    return Object.prototype.hasOwnProperty.call(event.requestContext, "elb");
   }
   return false;
 };
 var isProxyEventV2 = (event) => {
-  return Object.hasOwn(event, "rawPath");
+  return Object.prototype.hasOwnProperty.call(event, "rawPath");
 };
 var isContentTypeBinary = (contentType) => {
   return !/^(text\/(plain|html|css|javascript|csv).*|application\/(.*json|.*xml).*|image\/svg\+xml.*)$/.test(
diff --git a/dist/cjs/adapter/aws-lambda/handler.js b/dist/cjs/adapter/aws-lambda/handler.js
index 7a07e8c474550ce3cadc20f4309f7e15a40e0bb4..5628773c094a98c9a675396547be8e6a52d6d0b1 100644
--- a/dist/cjs/adapter/aws-lambda/handler.js
+++ b/dist/cjs/adapter/aws-lambda/handler.js
@@ -284,12 +284,12 @@ const getProcessor = (event) => {
 };
 const isProxyEventALB = (event) => {
   if (event.requestContext) {
-    return Object.hasOwn(event.requestContext, "elb");
+    return Object.prototype.hasOwnProperty.call(event.requestContext, "elb");
   }
   return false;
 };
 const isProxyEventV2 = (event) => {
-  return Object.hasOwn(event, "rawPath");
+  return Object.prototype.hasOwnProperty.call(event, "rawPath");
 };
 const isContentTypeBinary = (contentType) => {
   return !/^(text\/(plain|html|css|javascript|csv).*|application\/(.*json|.*xml).*|image\/svg\+xml.*)$/.test(
