diff --git a/dist/adapter/aws-lambda/handler.js b/dist/adapter/aws-lambda/handler.js
index e7cb73df408898ad699df3edc99e4f8d45385fe6..d760c0ac0d054e647f38245807ff5ed6404444c4 100644
--- a/dist/adapter/aws-lambda/handler.js
+++ b/dist/adapter/aws-lambda/handler.js
@@ -260,12 +260,12 @@ var getProcessor = (event) => {
 };
 var isProxyEventALB = (event) => {
   if (event.requestContext) {
-    return Object.hasOwn(event.requestContext, "elb");
+    return Object.prototype.hasOwnProperty.call(event.requestContext, "elb");
   }
   return false;
 };
 var isProxyEventV2 = (event) => {
-  return Object.hasOwn(event, "rawPath");
+  return Object.prototype.hasOwnProperty.call(event, "rawPath");
 };
 var isContentTypeBinary = (contentType) => {
   return !/^(text\/(plain|html|css|javascript|csv).*|application\/(.*json|.*xml).*|image\/svg\+xml.*)$/.test(
diff --git a/dist/cjs/adapter/aws-lambda/handler.js b/dist/cjs/adapter/aws-lambda/handler.js
index e423d099f33f355fcc598b43e5728e16d7a2284b..2538fbf7b47b6a03b2da1c24795f11861fa7f11e 100644
--- a/dist/cjs/adapter/aws-lambda/handler.js
+++ b/dist/cjs/adapter/aws-lambda/handler.js
@@ -290,12 +290,12 @@ const getProcessor = (event) => {
 };
 const isProxyEventALB = (event) => {
   if (event.requestContext) {
-    return Object.hasOwn(event.requestContext, "elb");
+    return Object.prototype.hasOwnProperty.call(event.requestContext, "elb");
   }
   return false;
 };
 const isProxyEventV2 = (event) => {
-  return Object.hasOwn(event, "rawPath");
+  return Object.prototype.hasOwnProperty.call(event, "rawPath");
 };
 const isContentTypeBinary = (contentType) => {
   return !/^(text\/(plain|html|css|javascript|csv).*|application\/(.*json|.*xml).*|image\/svg\+xml.*)$/.test(
