diff --git a/dist/adapter/aws-lambda/handler.js b/dist/adapter/aws-lambda/handler.js
index 85573985f261567ed1eef26d0c1278564b1aa237..f660a7bccfdf9e7ac845f7452d147bac9be18a3d 100644
--- a/dist/adapter/aws-lambda/handler.js
+++ b/dist/adapter/aws-lambda/handler.js
@@ -251,10 +251,10 @@ var getProcessor = (event) => {
   return v1Processor;
 };
 var isProxyEventALB = (event) => {
-  return Object.hasOwn(event.requestContext, "elb");
+  return Object.prototype.hasOwnProperty.call(event.requestContext, "elb");
 };
 var isProxyEventV2 = (event) => {
-  return Object.hasOwn(event, "rawPath");
+  return Object.prototype.hasOwnProperty.call(event, "rawPath");
 };
 var isContentTypeBinary = (contentType) => {
   return !/^(text\/(plain|html|css|javascript|csv).*|application\/(.*json|.*xml).*|image\/svg\+xml.*)$/.test(
diff --git a/dist/cjs/adapter/aws-lambda/handler.js b/dist/cjs/adapter/aws-lambda/handler.js
index e402dfc79ae9dc1d7c7bf7e6a159f61caeb04503..581c039f497ca34503139f17dcc59f5c8a7161ad 100644
--- a/dist/cjs/adapter/aws-lambda/handler.js
+++ b/dist/cjs/adapter/aws-lambda/handler.js
@@ -283,10 +283,10 @@ const getProcessor = (event) => {
   return v1Processor;
 };
 const isProxyEventALB = (event) => {
-  return Object.hasOwn(event.requestContext, "elb");
+  return Object.prototype.hasOwnProperty.call(event.requestContext, "elb");
 };
 const isProxyEventV2 = (event) => {
-  return Object.hasOwn(event, "rawPath");
+  return Object.prototype.hasOwnProperty.call(event, "rawPath");
 };
 const isContentTypeBinary = (contentType) => {
   return !/^(text\/(plain|html|css|javascript|csv).*|application\/(.*json|.*xml).*|image\/svg\+xml.*)$/.test(
